### PR TITLE
Fix a bug `transform_matrix_offset_center`

### PR DESCRIPTION
**Pull request by 2wins**
- calculation of offset: sign error

### Checklist
- [x] I've tested that my changes are compatible with the latest version of Tensorflow.
- [x] I've read the [Contribution Guidelines](https://github.com/tensorlayer/tensorlayer/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### CODE REVIEW

**Comment by DEKHTIARJonathan** [comment]
@zsdonghao if it is good to you, you can approve & merge ;)

File: tensorlayer/prepro.py [hunk 1]
```diff
@@ -1719,8 +1719,8 @@ def transform_matrix_offset_center(matrix, x, y):
     - See ``tl.prepro.rotation``, ``tl.prepro.shear``, ``tl.prepro.zoom``.
 
     """
-    o_x = float(x) / 2 + 0.5
-    o_y = float(y) / 2 + 0.5
+    o_x = float(x) / 2 - 0.5
```
Comment (lgarithm) [inline tensorlayer/prepro.py]: could you use 
```
o_x = (x - 1) / 2.0
o_y = (y - 1) / 2.0
```

to make it more readable?

Comment (2wins) [reply tensorlayer/prepro.py]: OK. I'll change it.

File: tensorlayer/prepro.py [hunk 2]
```diff
@@ -1719,8 +1719,8 @@ def transform_matrix_offset_center(matrix, x, y):
     - See ``tl.prepro.rotation``, ``tl.prepro.shear``, ``tl.prepro.zoom``.
 
     """
-    o_x = float(x) / 2 + 0.5
-    o_y = float(y) / 2 + 0.5
+    o_x = (x - 1) / 2
```
Comment (lgarithm) [inline tensorlayer/prepro.py]: In python2, it doesn't work.
Still need `(x - 1) / 2.0`

Comment (2wins) [reply tensorlayer/prepro.py]: Ah, it's my mistake!

**Pull request merged**